.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,31 @@
+name: build_and_test
```

Comment:
You commit message misses some letters. `iplement` -> `implement`, `ation` -> `action`. Regarding the commit content - I would propose to get a review on that from somebody who knows that stuff. I have no idea how our CI works.

Reply:
Fixed commit message, i show this patch to Alexandr Tikhonov after you